matador/plotting/ir_plotting.py

```diff
@@ -0,0 +1,126 @@
+
+
+def read_ir_file(seed):
+    """ This function should take the seed filename and
+    read in the IR data, probably into a dictionary.
+
+    Parameters:
+        seed (str): the filename to read.
+
+    Returns:
+        dict: containing IR data
+
+    """
+
+    with open(seed + '.phonon', 'r') as f:
+        flines = f.readlines()
+    for ind, line in enumerate(flines):
+        if 'Number of branches' in line:
+            no_branches = int(line.split()[3])
+        if 'END header' in line:
+            header_end = ind
+
+    # contains IR and Raman data
+    ir_dat = dict()
+    for line in flines[header_end+2:header_end+no_branches+2]:
+        line_split = line.split()
+        if len(line_split) < 2:
+            print_failure('Failed to read IR data. Check .phonon file.')
+            exit()
+        wavenumber = float(line_split[1])
+        ir = float(line_split[2])
+        # checks for raman data
+        if len(line_split) > 3:
+            raman = float(line_split[3])
+            if wavenumber in ir_dat:
+                ir_dat[wavenumber] = [ir_dat[wavenumber][0]+ir, ir_dat[wavenumber][1]+raman]
+            else:
+                ir_dat[wavenumber] = [ir, raman]
+        else:
+            if wavenumber in ir_dat:
+                ir_dat[wavenumber] = [ir_dat[wavenumber][0]+ir]
+            else:
+                ir_dat[wavenumber] = [ir]
+
+    return(ir_dat)
+
+# this decorator will automatically set the style of the plot
+# to match the user's config/matador defaults, and safely handles
+# e.g. X-forwarding and file writing
+@plotting_function
+def plot_ir_spectrum(seed, ir_ss, ir_bs):
+    """ This function plots the IR spectrum found in the given file.
+
+        Parameters:
+        bin_width: The width of the wavenumber bin in cm^-1. Determined by step_size and bin_scaler
+
+        Keyword Arguments:
+        step_size: change in wavenumber between points on x-axis,
+        bin_scaler: scaled to the step_size (1 is minimum vale); used for gaussian broadening,
+    """
+    import matplotlib.pyplot as plt
+
+    step_size = ir_ss
+    bin_scaler = ir_bs
+    bin_width = step_size * bin_scaler
+
+    # read IR output
+    ir_dat = read_ir_file(seed)
+    wavenumbers_castep = list(ir_dat.keys())
+    if len(ir_dat[wavenumbers_castep[0]]) == 2:
+        raman = True
+    else:
+        raman = False
+    min_wavenumber = int(wavenumbers_castep[0])-(2*step_size)
+    max_wavenumber = int(wavenumbers_castep[len(wavenumbers_castep)-1])+(2*step_size)
+    wavenumbers_plot = []
+    for i in range(min_wavenumber, max_wavenumber, step_size):
+        wavenumbers_plot.append(i)
+
+    ir_plot = []
+    raman_plot = []
+    for i in wavenumbers_plot:
+        ir_intensity = 0
+        raman_intensity = 0
+        for j in wavenumbers_castep:
+            if j >= (i-(0.5*bin_width)) and j < (i+(0.5*bin_width)):
+                ir_intensity = ir_intensity + ir_dat[j][0]
+                if raman == True:
```

Comment:
```suggestion
                if raman:
```

Can replace all these `== True` statements similarly